src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java

```diff
@@ -92,10 +96,29 @@ public class CaseData {
 
     @CCD(
         label = "Case name Hmcts Internal",
-        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class, GlobalSearchAccess.class}
     )
     private String caseNameHmctsInternal;
 
+    @CCD(
+        label = "Search Criteria",
+        access = {GlobalSearchAccess.class}
+    )
+    @SuppressWarnings("MemberName") // Field name is case-sensitive in CCD
```

Comment:
Comment is good for clarity thank you, but is it still required on merging?

Reply:
Yeah, better to keep this comment as the dev will know why the case is different for this field and they won't update in the future in case of any refinement.